fdbserver/MasterProxyServer.actor.cpp

```diff
@@ -1191,14 +1191,23 @@ struct TransactionRateInfo {
 	}
 };
 
-ACTOR Future<Void> sendGrvReplies(Future<GetReadVersionReply> replyFuture, std::vector<GetReadVersionRequest> requests, ProxyStats *stats) {
+ACTOR Future<Void> sendGrvReplies(Future<GetReadVersionReply> replyFuture, std::vector<GetReadVersionRequest> requests,
+                                  ProxyStats* stats, Version minKnownCommittedVersion) {
 	GetReadVersionReply reply = wait(replyFuture);
+
+	GetReadVersionReply minKCVReply = reply;
+	minKCVReply.version = minKnownCommittedVersion;
```

Comment:
move this copy inside the loop to avoid copying when requests do not have this flag